everestrs/everestrs/build.rs

```diff
@@ -7,6 +7,10 @@ struct Libraries {
 }
 
 fn find_everest_workspace_root() -> PathBuf {
+    if let Ok(everest_framework_dir) = env::var("EVEREST_RS_FRAMEWORK_SOURCE_LOCATION") {
```

Comment:
The naming is a bit off here, is it everest framework dir OR everestrs framework source location?

Reply:
My intention with naming here was that this environment variable is meant for everestrs and contains the location to the source of everest-framework